airflow/models/dag.py

```diff
@@ -2181,9 +2199,9 @@ def clear(
         )
 
         if dry_run:
-            return tis
+            return session.scalars(tis).all()
```

Comment:
Need to update the return type annotation of this function to match

Reply:
Looking at it more, it seems the return type matches. Can you verify?